serverless/networking/serverless-ossm.adoc

```diff
@@ -0,0 +1,126 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm"]
+= Using {ProductShortName} with {ServerlessProductName}
+:context: serverless-ossm
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// what benefits does this integration provide?
+
+.Prerequisites
+. Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+. Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+
+.Procedure
+. Add the `default` namespace to the xref:../../service_mesh/service_mesh_install/installing-ossm.adoc#ossm-member-roll-create_installing-ossm[ServiceMeshMemberRoll] as a member:
++
+
+[source,yml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    - default
+----
++
+[IMPORTANT]
+====
+Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
```

Comment:
It is not supported for now, but it will be supported (probably we will). So can you change the impression? :smile: